packages/components/src/styles/global.js

```diff
@@ -1,22 +1,26 @@
 import { GlobalStyles } from "twin.macro";
 import React from "react";
+import arimoBoldWoff from "./fonts/arimo-bold.woff";
```

Comment:
I think for export purposes, we probably don't want this part in our components exports, but instead as a step in our README. 

That or we need to expect some level of loader for this file, maybe that's fine.